src/scenes/AcceleratorRouter/ParticipantProjects/EditView/index.tsx

```diff
@@ -58,6 +62,11 @@ const EditView = () => {
 
   const [confirmProjectNameModalOpen, setConfirmProjectNameModalOpen] = useState(false);
 
+  const projectApplication = useMemo(
+    () => allApplications?.find((app) => app.projectId === projectId),
+    [allApplications, projectId]
+  );
+
```

Comment:
I'm not sure if there's a more convenient method available to get an application for a given project?